cypress/integration/order-history.test.js

```diff
@@ -0,0 +1,40 @@
+describe('empty order history test', () => {
+  const MOCK_EMAIL = 'kapov@gmail.com';
+  const MOCK_PASSWORD = 'QazxsW21';
+  it('should enter and open order history page', () => {
+    cy.visit('/login');
+    cy.get(
+      '.makeStyles-emailInput-487 > .MuiInputBase-root > .MuiInputBase-input'
+    )
+      .should('be.visible')
+      .type(MOCK_EMAIL);
+    cy.get(
+      '.makeStyles-passwordInput-486 > .MuiInputBase-root > .MuiInputBase-input'
```

Comment:
Class names created by `makeStyles` are not static, use the `data-cy` attribute to select element

Reply:
this problem has been solved